ketama/ketama_test.go

```diff
@@ -31,11 +32,49 @@ func TestContinuumDistribution(t *testing.T) {
 
 	// Value should be roughly equal to 10000 / num_servers
 	target := float64(10000) / float64(len(serverMap))
-	errorRange := float64(target * .18)
+	errorRange := float64(target * .20)
```

Comment:
reverted; I originally bumped to keep consistent w/ the second test which didn't have as even hash distribution